packages/docs/docs/self-hosting/config-settings.md

```diff
@@ -135,7 +135,6 @@ You will also be prompted for a parameter "Type". The default option is "String"
 | `DatabaseSecrets`      | The AWS Secret ID containing database connection details (created automatically by CDK). Only available when using AWS Parameter Store config. See [AWS Secrets](#aws-secrets).                                                                                                   |          | `cdk`      |                                     |
 | `redis`                | The redis connection details as a JSON object. Only available when using JSON config file. See [AWS Secrets](#aws-secrets).                                                                                                                                                       |          |            |                                     |
 | `RedisSecrets`         | The AWS Secret ID containing Redis connection details (created automatically by CDK). Only available when using AWS Parameter Store config. See [AWS Secrets](#aws-secrets).                                                                                                      |          | `cdk`      |                                     |
-| `logRequests`          | Optional flag to log individual HTTP requests.                                                                                                                                                                                                                                    |          |            | `false`                             |
```

Comment:
I think we should keep the config setting around, and just flip the switch in our prod environments.

This feature is very noisy on localhost / dev mode.